Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -442,29 +442,19 @@ public final class ManifestLoader: ManifestLoaderProtocol {
         completion: @escaping (Result<EvaluationResult, Error>) -> Void
     ) {
         do {
-            if localFileSystem.isFile(manifestPath) {
```

Comment:
do we still need the `manifestPath` parameter?

Reply:
nope

Reply:
Actually, we do need it since this method is the only place where we know both the temporary path and the original one so that we can do the replacement in compiler output.